docs/users/cli-usage.md

```diff
@@ -5,12 +5,12 @@ Type `ddev` or `ddev -h`in a terminal windows to see the available ddev commands
 
 ## Quickstart Guides
 
-You can start using ddev by running just a few commands. Below are quickstart instructions WordPress, Drupal 7, Drupal 8, and TYPO3 CMS.
+These are quickstart instructions WordPress, Drupal 7, Drupal 8, TYPO3, and Backdrop.
 
-**Prerequisites:** If you do not have ddev already on your machine, please follow the [installation instructions](../index.md#installation) before beginning the quickstart tutorials.  You'll need *docker* and *docker-compose* to use ddev.
+**Prerequisites:** Before you start, [check the system requirements](../index.md#system-requirements) and follow the [installation instructions](../index.md#installation).
```

Comment:
We deliberately added the piece mentioning docker and docker-compose because people were skipping by it.

Reply:
+1 regarding keeping docker-compose

Reply:
Ah okey! Will emphasize this and fix.